your_daily_dose_malware/app/utils/http_utils.py

```diff
@@ -0,0 +1,65 @@
+from enum import Enum
+
+import requests
+from requests.exceptions import HTTPError, RequestException
+
+
+class HttpMethod(Enum):
+    GET = 'GET'
+    POST = 'POST'
+    PUT = 'PUT'
+    DELETE = 'DELETE'
+
+
+class HttpUtilsException(Exception):
+    """Custom exception class for HTTP utility errors."""
+
+    def __init__(self, message, status_code=None):
+        super().__init__(message)
+        self.status_code = status_code
+
+
+class HttpUtils:
```

Comment:
Im not sure why we might need an HTTPUtils class wrapper for this, the modules are just one call to a very similar module. Maybe its better if we write a s scrript to download and collect samples from one source, and we see what kind of structure we need organically and abstract it for more sources.